src/containers/Application.js

```diff
@@ -122,61 +124,37 @@ class Application extends Component {
 		else return (
 			<Provider store={this.Store}>
 				<div className="app">
-
-					<Switch>
-						{this.Router.Routes.map((route, idx) => {
-							return route.component ? (
-								<Route
-									key={idx}
-									path={`${route.path}`}
-									exact={route.exact}
-									name={route.name}
-									render={props => (
-										<React.Fragment>
-											{(route.hasHeader == true || route.hasHeader == undefined) ? (
-											<AppHeader fixed>
-												<Header app={this}/>
-											</AppHeader>
-											) : null}
-
-											<div className="app-body">
-												{(route.hasSidebar == true || route.hasSidebar == undefined) ? (
-												<AppSidebar fixed display="lg">
-													<AppSidebarNav
-														navConfig={this.Navigation.getItems()}
-														{...this.props}
-													/>
-													<AppSidebarFooter />
-													<AppSidebarMinimizer />
-												</AppSidebar>
-												) : null}
-
-												<main className="main">
-													{(route.hasBreadcrumb == true || route.hasBreadcrumb == undefined) ? (
-													<AppBreadcrumb appRoutes={this.Router.Routes}/>
-													) : null}
+					<Header app={this}/>
+					<div className="app-body">
+						<Sidebar app={this} navigation={this.Navigation}/>
+						<main className="main">
+							<AppBreadcrumb appRoutes={this.Router.Routes} router={router}/>
+							<Container fluid>
+								<Switch>
+									{this.Router.Routes.map((route, idx) => {
+										return route.component ? (
+											<Route
+												key={idx}
+												path={`${route.path}`}
+												exact={route.exact}
+												name={route.name}
+												render={props => (
 													<route.component app={this} {...props} {...route.props} />
-												</main>
-
-												<AppAside fixed>
-												</AppAside>
-											</div>
-
-											{(route.hasFooter == true || route.hasFooter == undefined) ? (
-											<AppFooter>
-												{this.props.footer ? this.props.footer : "Powered by ASAB"}
-											</AppFooter>
-											) : null}
-										</React.Fragment>
-									)}
-								/>
-							) : (null);
-						})}
-					</Switch>
+												)}
+											/>
+										) : (null);
+									})}
+									(this.DefaultPath != undefined ? <Redirect from="/" to={this.DefaultPath} />: null)
```

Comment:
Instead of parenthesis should be brackets

Reply:
Fixed